croud/config.py

```diff
@@ -22,35 +22,59 @@
 import yaml
 from appdirs import user_config_dir
 
+from croud.util import nested_dict_value
+
 
 class Configuration:
     USER_CONFIG_DIR: str = user_config_dir("Crate")
     FILENAME: str = "croud.yaml"
     FILEPATH: str = f"{USER_CONFIG_DIR}/{FILENAME}"
+    DEFAULT_CONFIG: dict = {
+        "auth": {"currentContext": 0, "contexts": [{"env": "prod", "token": ""}]}
+    }
 
     @staticmethod
     def create() -> None:
         if not os.path.exists(Configuration.USER_CONFIG_DIR):
             os.makedirs(Configuration.USER_CONFIG_DIR)
 
         if not os.path.isfile(Configuration.FILEPATH):
-            write_config({"env": "prod", "token": ""})
+            write_config(Configuration.DEFAULT_CONFIG)
 
     @staticmethod
     def get_env() -> str:
-        return load_config().get("env") or "prod"
+        return get_auth_context().get("env") or "prod"
```

Comment:
Why don't we do this?
```suggestion
        return get_auth_context().get("env", "prod")
```